portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,35 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a favorite quote to the page.
```

Comment:
It looks like you still display a "random" quote, so I would recommend changing this to "random favorite quote"